src/languages/en.js

```diff
@@ -306,4 +306,8 @@ export default {
     "One record": "One record",
     steamApiKeyDescription: "For monitoring a Steam Game Server you need a Steam Web-API key. You can register your API key here: ",
     "Current User": "Current User",
+    deleteProxyMsg: "Are you sure want to delete this proxy for all monitors?",
+    proxyDescription: "Proxies must be assigned to a monitor to function.",
```

Comment:
to monitor or to function (missing `or`?)

Reply:
This is based from notification description. It is tries to explain with "must be assigned to a monitor to function" the proxy does not effect until assigned to a monitor, so I think this should be stay same but if it is not descriptive enough sure we could update.